docusaurus/docs/ruby/hooks.mdx

```diff
@@ -42,75 +40,8 @@ KnapsackPro::Hooks::Queue.after_queue do |queue_id|
 end
 ```
 
-## RSpec
```

Comment:
We need to remove `If you are using RSpec or` from the caution above.

Reply:
I removed the caution. I think it does not make sense to keep it for percy because we recommend the default percy config as in percy docs. I removed legacy percy version from our docs because it was very old example.

https://github.com/KnapsackPro/docs.knapsackpro.com/pull/152/commits/4677f0fb157fabf082c4bff98887f395b3a0ca75